cli-tests/src/upload.rs

```diff
@@ -235,14 +236,31 @@ async fn upload_bundle_using_bep() {
         .file_sets
         .iter()
         .for_each(|file_set| {
-            assert_eq!(file_set.resolved_status, Some(JunitReportStatus::Passed));
             assert_eq!(file_set.file_set_type, FileSetType::Junit);
             let mut junit_parser = JunitParser::new();
             file_set.files.iter().for_each(|file| {
                 let junit_file = fs::File::open(tar_extract_directory.join(&file.path)).unwrap();
                 assert!(junit_parser.parse(BufReader::new(junit_file)).is_ok());
                 assert!(junit_parser.issues().is_empty());
             });
+            let report = junit_parser.into_reports().pop().unwrap();
+            let test_runner_report = file_set.test_runner_report.unwrap();
+            assert_eq!(
+                test_runner_report.resolved_status,
+                TestRunnerReportStatus::Passed
+            );
+            assert!(
+                (test_runner_report.resolved_start_time_epoch_ms
+                    - report.timestamp.unwrap().to_utc())
+                .abs()
+                    <= TimeDelta::milliseconds(1)
+            );
+            assert!(
+                (test_runner_report.resolved_end_time_epoch_ms
+                    - (report.timestamp.unwrap().to_utc() + report.time.unwrap()))
+                .abs()
+                    <= TimeDelta::milliseconds(1)
+            );
```

Comment:
I split these out to prevent any issues with small nanosecond rounding issues when roundtripping gRPC timstamps